src/CHANGES.txt

```diff
@@ -6,6 +6,10 @@
 
 RELEASE  VERSION/DATE TO BE FILLED IN LATER
 
+  From Philipp Maierhöfer:
+    - Avoid crash with UnicodeDecodeError on Python 3 when a Latex log file in
+      non-UTF-8 encoding (e.g. containing umlauts in Latin-1 encoding) is read.
```

Comment:
Should this also say something about "\usepackage[T1]{fontenc}" type syntax being used?

Reply:
Ok, done. Also added comment about the test case and compile only to DVI, not to PostScript (that's sufficient).